content/en/docs/setup/install/multicluster/virtual-machine/index.md

```diff
@@ -0,0 +1,156 @@
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/root-cert.pem \
+        --from-file="${HOME}"/"${CLUSTER_NAME}"/cert-chain.pem
+    {{< /text >}}
+
+1. Generate a `cluster.env` configuration file that informs the virtual machine
+   deployment which network CIDR to capture and redirect to the Kubernetes
+   cluster:
+
+    {{< text bash >}}
+    $ ISTIO_SERVICE_CIDR=$(echo '{"apiVersion":"v1","kind":"Service","metadata":{"name":"tst"},"spec":{"clusterIP":"1.1.1.1","ports":[{"port":443}]}}' | kubectl apply -f - 2>&1 | sed 's/.*valid IPs is //')
+    $ echo -n ISTIO_SERVICE_CIDR=$ISTIO_SERVICE_CIDR > "${HOME}"/"${CLUSTER_NAME}"/cluster.env
+    {{< /text >}}
+
+1. Configure a select set of ports for exposure from the virtual machine. If you do not
+   apply this optional step, all outbound traffic on all ports is sent to the
+   Kubernetes cluster. You may wish to send some traffic on specific ports to
+   other destinations. This example shows enabling ports `3306` and `8080`
+   for capture by Istio virtual machine integration and transmission to Kubernetes.
+   All other ports are sent over the default gateway of the virtual machine.
+
+    {{< text bash >}}
+    $ echo "ISTIO_INBOUND_PORTS=3306,8080" >> "${HOME}"/"${CLUSTER_NAME}"/cluster.env
+    {{< /text >}}
+
+1. Is this step needed if we always assume there is a load balancer? Try removing.
+    The ingress gateway port for Istiod is automatically programmed to 15012 throughout
+    the Istio software. Record the ISTIO_PILOT_PORT address in cluster.env.
+
+    {{< text bash >}}
+    $ echo "ISTIO_PILOT_PORT=15012" >> ${HOME}/${CLUSTER_NAME}"/cluster.env
+    {{< /text >}}
+
+
+1. Install Istio with virtual machine integration features enabled:
+
+    {{< text bash >}}
+    $ cat <<EOF> vmintegration.yaml
+    apiVersion: install.istio.io/v1alpha1
+
+    kind: IstioOperator
+    spec:
+      values:
+        global:
+          meshExpansion:
+            enabled: true
+          controlPlaneSecurityEnabled: true
+    EOF
+    $ istioctl install -f vmintegration.yaml
+    {{< /text >}}
+
+### Installing the virtual machine
+
+Run the following commands on the virtual machine you want to add to the Istio mesh:
+
+1. Copy the previously created `cluster.env` and `*.pem` files to the virtual machine.
+
+1. Install the Debian package containing the Istio virtual machine integration runtime.
+
+    {{< text bash >}}
+    $ curl -LO https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/deb/istio-sidecar.deb
+    $ sudo dpkg -i istio-sidecar.deb
+    {{< /text >}}
+
+1. Add an IP address for Istiod to `/etc/hosts`. Replace `<INGRESS_GATEWAY>` with the
+    ingress gateway service of istiod. Revisit
+    [Determining the Ingress IP and ports](/docs/tasks/traffic-management/ingress/ingress-control/#determining-the-ingress-ip-and-ports) to determine the <INGRESS_GATEWAY>.
+
+    {{< text bash >}}
+    $ sudo echo "<INGRESS_GATEWAY> istiod.istio-system.svc" >> /etc/hosts
```

Comment:
Use `$INGRESS_HOST`? which is set in the previous doc

Reply:
SGTM.

Reply:
addressed. One problem is if the load balancer uses DNS, the previous doc does not work with this doc. I am not immediately sure how to rectify this problem.